src/Conversation.res

```diff
@@ -54,31 +58,33 @@ let make = (
 
   let scrollUp = () =>
     switch scrollableRef.current->Js.Nullable.toOption {
-    | Some(domNode) =>
-      scrollElementToTop(domNode)->ignore
+    | Some(domNode) => scrollElementToTop(domNode)->ignore
     | None => ()
     }
 
   //TODO: Scroll to top when a reply has been sent
   let (state, send) = ReactUpdate.useReducer((state, action) =>
     switch action {
     | ToggleNotes => ReactUpdate.Update({...state, show_notes: !state.show_notes})
-    | NotesChanged(notes) => ReactUpdate.Update({...state, notes: notes})
+    | NotesChanged(notes) => ReactUpdate.Update({...state, notes})
     }
   , initialState)
 
   <div
-    className={list{
```

Comment:
We should generally ignore the `list` data structure and convert everything into arrays. Arrays are way easier to handle from a JS perspective, unless you really need the pattern matching capabilities (90% not the case).

Also for the `app.res` I'd recommend to convert most data structures from `list` to `array` and use the proper core apis for `Array`.